platform/docs/docs/platform/managers/service.md

```diff
@@ -86,9 +87,16 @@ export default {
 and the implementation of `ToolBarService` lies in the same folder at
 `./ToolbarSerivce.js`.
 
-> Note, the create method is critical for any custom service that you write and
+> Note: The create method is critical for any custom service that you write and
 > want to add to the list of services
 
+> Note: For typescript definitions, the service type should be exported
+> as part of the Types export on the module.  This is recommended going forward
+> and existing services will be migrated.  As well, the capitalization of the
+> name should be lower camel case, with the type being upper camel case.  In
+> the above example, the service instance should be `toolBarService` with the
+> class being `ToolBarService`.
```

Comment:
Are you proposing to change all the services from capitalized to lowercase? why?

Reply:
I'm eventually proposing that we do it using our specified naming convention, which is:
Type names are UpperCamelCase
instance names are lowerCamelCase
and one does not use the Type name as the instance name - they can be upper/lower camel case equivalent, but they don't overlap.  That prevents confusion when importing actual types.

The fact that we have lots of confusion over the naturalized DICOM data names is because the names should have gotten correctly capitalized.

My proposal is that as we add Type exports, we add a historical upper-case alias, and switch the canonical name to lowerCamelCase.